jenkinsapi/jenkins.py

```diff
@@ -222,6 +223,10 @@ def delete_view_by_url(self, str_url):
         self.poll()
         return self
 
+    def get_label_by_url(self, label_url):
```

Comment:
There is no API to get all labels, AFAIK, but all labels are accessible via `http://jenkins:port/label/label_name`. Why not have `get_label` function, that will return `Label` object by label name?